### DISCUSSION TITLE
Font size during typing text instead of size of tool

### DISCUSSION THREAD

**Discussion by gryznar** [flameshot-org/flameshot#2969] [Ideas]
It would be great if there was an option to set the font size of the text being typed in, instead of operating on a slider. The current behavior is not quite user-friendly (it is not possible to change the font on the already selected area, the text size has to be selected by trial and error.

**Comment by mmahmoudian** [comment]
you can change the font size or tool size by any of the following approaches **when the tool is selected** :
1. scrolling your mouse wheel
2. opening the sidebar (for example by pressing <kbd>Space</kbd>) and use the slider
3. opening the sidebar (for example by pressing <kbd>Space</kbd>) and type the number in the text box near the slider
4. type the number (when the tool is selected)

I hope this quick video helps:

[Peek 2022-11-03 15-18.webm](https://user-images.githubusercontent.com/390889/199731101-55de481b-8c4d-4669-8f5e-18b2c6002c64.webm)

**Reply by gryznar** [reply]
Thanks!
